scripts/release.sh

```diff
@@ -37,6 +37,17 @@ specified branch as the release commit. This will also set --dry-run.
 EOH
 }
 
+# Temporary workaround for commits on `main` without a PR, we can remove this
+# once we ensure all commits for a release will have a PR.
+# Also see .github/workflow/release.yml.
+export CODER_IGNORE_MISSING_COMMIT_METADATA=1
```

Comment:
We probably don't need this since we can just set it manually ourselves, less to remember to manually undo later on.

Reply:
Hmm.. good idea! Then I'll just turn this into an action input 🤔. Then we can default to require metadata.